packages/client/src/components/Login.re

```diff
@@ -119,10 +120,11 @@ let make = (~onLogin) => {
         <input
```

Comment:
I know this is not a part of this PR and I can do it after it gets merged, but I think using a `<button>`-element here would be nice so one doesn't have to look for the `type` to see that this is submitting!